label.py

```diff
@@ -11,80 +11,106 @@
+cv2.namedWindow('firefox')
+cv2.namedWindow('chrome')
+
+
+def draw_boundary_box(event, mouse_x, mouse_y, flags, param):
+    global start_x, start_y, drawing, end_x, end_y
+    [drawing_area, boxes] = param
+
+    if event == cv2.EVENT_LBUTTONDOWN:
+        drawing = True
+        start_x, start_y = mouse_x, mouse_y
+        end_x, end_y = mouse_x, mouse_y
+    elif event == cv2.EVENT_MOUSEMOVE:
+        if drawing is True:
+            box_start_x, box_start_y = start_x, start_y
+            box_end_x, box_end_y = end_x, end_y
+            if box_start_x > box_end_x:
+                box_start_x, box_end_x = box_end_x, box_start_x
+            if box_start_y > box_end_y:
+                box_start_y, box_end_y = box_end_y, box_start_y
+            box_start_x = max(0, box_start_x)
+            box_start_y = max(0, box_start_y)
+            box_end_x = min(drawing_area.shape[1], box_end_x)
+            box_end_y = min(drawing_area.shape[0], box_end_y)
+            drawing_area[box_start_y:box_end_y + 1, box_start_x:box_end_x + 1, 0:3] = 0
+            end_x = max(mouse_x, 0)
+            end_y = max(mouse_y, 0)
+            cv2.rectangle(drawing_area, (start_x, start_y), (end_x, end_y), (0, 255, 0), -1)
+    elif event == cv2.EVENT_LBUTTONUP:
+        drawing = False
+        start_x = max(0, start_x)
+        start_y = max(0, start_y)
+        end_x = min(drawing_area.shape[1], end_x)
+        end_y = min(drawing_area.shape[0], end_y)
+        cv2.rectangle(drawing_area, (start_x, start_y), (end_x, end_y), (0, 255, 0), -1)
+        boxes.append([min(start_x, end_x), min(start_y, end_y), max(start_x, end_x), max(start_y, end_y)])
 
 
 def get_new_image():
-    global current_image
     if len(images_to_show) == 0:
-        root.quit()
+        cv2.destroyAllWindows()
         return
     current_image = images_to_show.pop()
-    print("data/%s_firefox.png" % current_image)
-
-    img_firefox = Image.open("data/%s_firefox.png" % current_image)
-    img_chrome = Image.open("data/%s_chrome.png" % current_image)
-
-    img = ImageTk.PhotoImage(img_firefox)
-    panel1.configure(image=img)
-    panel1.image = img
-    img = ImageTk.PhotoImage(img_chrome)
-    panel2.configure(image=img)
-    panel2.image = img
-
-    background = img_firefox.convert("RGBA")
-    overlay = img_chrome.convert("RGBA")
-    img_blended = Image.blend(background, overlay, 0.5)
-
-    img = ImageTk.PhotoImage(img_blended)
-    panel3.configure(image=img)
-    panel3.image = img
-
-
-# The images are the same.
-def callback_y(e):
-    labels[current_image] = 'y'
-    get_new_image()
-
-
-# The image are basically the same, except for advertisement or content.
-def callback_d(e):
-    labels[current_image] = 'd'
+    print("Reading %s" % current_image)
+    firefox_ss = cv2.imread("data/%s_firefox.png" % current_image)
+    chrome_ss = cv2.imread("data/%s_chrome.png" % current_image)
+    drawing_area_firefox = np.zeros((firefox_ss.shape), np.uint8)
```

Comment:
Put this stuff in a `reset_bounding_boxes` function, so you don't have to repeat the same code here and when `r` is presseed